### PR TITLE
feat: update website to use gatsby-carbon-theme

### PR DESCRIPTION
**Pull request by alisonjoseph**
### WIP

Complete refactor to use the `gatsby-carbon-theme` as the codebase

## Testing

Compare to www.carbondesignsystem.com

### CODE REVIEW

**Comment by vpicone** [comment]
WhAt in tarnation happened here

**Comment by vpicone** [comment]
Are you okay

**Comment by alisonjoseph** [comment]
> Are you okay

Nope

> WhAt in tarnation happened here

trying to get CircleCI to work 😭

**Comment by alisonjoseph** [comment]
image

**Comment by sidharthachatterjee** [comment]
@alisonjoseph Can you try the `circleci/node10` image for CircleCI? And see if you can bump up memory on the instances? We build gatsbyjs.org with that image and `GATSBY_CPU_COUNT=8` env

File: .circleci/config.yml [hunk 1]
```diff
@@ -11,15 +11,12 @@ jobs:
...
-      - run:
-          name: Run Continuous Integration checks
-          command: PATH=$HOME/.yarn/bin:$PATH yarn ci-check
+          name: Install dependencies
+          command: PATH=$HOME/.yarn/bin:$PATH yarn install
       - run:
           name: Confirm project can build
           command: |
-            PATH=$HOME/.yarn/bin:$PATH yarn build:external && yarn storybook:build
+            PATH=$HOME/.yarn/bin:$PATH yarn build
```
Comment (DSchau) [inline .circleci/config.yml]: Try adding `GATSBY_CPU_COUNT=2` (or 1) to hard-code the amount of CPUs available to the container.

CircleCI unfortunately reports more than are actually available (I think 8 or something) which then leads to some OOM issues.

Feel free to reach out on Twitter if I can help more; thanks!

Comment (sidharthachatterjee) [reply .circleci/config.yml]: Thanks Dustin!

**Comment by sidharthachatterjee** [comment]
You might want to hard code it to `GATSBY_CPU_COUNT=2` or `GATSBY_CPU_COUNT=1` if you have the default instances on CircleCI

We set it to `GATSBY_CPU_COUNT=8` for the xxlarge ones I think

**Pull request closed without merging**